tests/openapi/test_request_body.py

```diff
@@ -28,3 +28,28 @@ def test_create_request_body() -> None:
             if "data" in handler_fields:
                 request_body = create_request_body(field=handler_fields["data"], generate_examples=True, plugins=[])
                 assert request_body
+
+
+def test_upload_file_request_body() -> None:
+    app = Starlite(route_handlers=[handle_file_upload])
+    assert app.openapi_schema.dict(exclude_none=True)["components"] == {  # type: ignore[union-attr]
```

Comment:
@cofin please get me a dump of the components generated on your end like the above